Fix #18995

corsonr opened the pull request:
Fixes #18995 by moving the note section outside the addresses container and unlink it from the shipping method conditional statement.

![](http://cld.wthms.co/T8qe0M+)

corsonr committed:
Fix #18995

Fixes #18995 by moving the note section outside the addresses container and unlink it from the shipping method conditional statement.

mikejolley commented:
This shows 'note' when there is no note. I'd suggest leaving it's conditional wrapping, and leaving it in column 2 outside of the conditional seeing if there is a shipping address. That way it also won't need more styling.

corsonr committed:
Do not display H2 note if no note

This will make the note title not appear if no note is defined. However, some CSS styling is needed as padding was originally added via .wc-order-preview-address wichi was containing the note section.

corsonr commented:
Ok, I updated the code, however, it seems that using a dedicated class for the note div is needed to fix padding issues.
